### PR TITLE
feat: use cloudflare secrets for edge workers demo

### PR DESCRIPTION
**Pull request by bruuuuuuuce**

### CODE REVIEW

File: examples/cloudflare-workers/wrangler.toml [hunk 1]
```diff
@@ -2,10 +2,6 @@ name = "momento-cloudflare-worker-http"
...
 
-# Variable bindings. These are arbitrary, plaintext strings (similar to environment variables)
-# Note: Use secrets to store sensitive data.
-# Docs: https://developers.cloudflare.com/workers/platform/environment-variables
 # [vars]
-# MOMENTO_AUTH_TOKEN = "REPLACE ME"
-# MOMENTO_REST_ENDPOINT = "REPLACE ME"
-# MOMENTO_CACHE_NAME = "REPLACE ME"
+# MOMENTO_REST_ENDPOINT = "https://api.cache.cell-4-us-west-2-1.prod.a.momentohq.com"
+# MOMENTO_CACHE_NAME = "w00t"
```
Comment (cprice404) [inline examples/cloudflare-workers/wrangler.toml:7]: did you mean to commit this

Comment (bruuuuuuuce) [reply examples/cloudflare-workers/wrangler.toml:7]: o no i can remove it and leave it as `REPLACE ME`

File: examples/cloudflare-workers/wrangler.toml [hunk 2]
```diff
@@ -2,10 +2,6 @@ name = "momento-cloudflare-worker-http"
...
 compatibility_date = "2023-07-10"
 
-# Variable bindings. These are arbitrary, plaintext strings (similar to environment variables)
-# Note: Use secrets to store sensitive data.
-# Docs: https://developers.cloudflare.com/workers/platform/environment-variables
 # [vars]
-# MOMENTO_AUTH_TOKEN = "REPLACE ME"
-# MOMENTO_REST_ENDPOINT = "REPLACE ME"
-# MOMENTO_CACHE_NAME = "REPLACE ME"
+# MOMENTO_REST_ENDPOINT = "https://api.cache.cell-4-us-west-2-1.prod.a.momentohq.com"
```
Comment (bruuuuuuuce) [inline examples/cloudflare-workers/wrangler.toml:6]:
```suggestion
# MOMENTO_REST_ENDPOINT = "REPLACE ME"
```

File: examples/cloudflare-workers/wrangler.toml (hunk 1 above)
Comment (bruuuuuuuce) [inline examples/cloudflare-workers/wrangler.toml:7]:
```suggestion
# MOMENTO_CACHE_NAME = "REPLACE ME"
```

**Review by cprice404 — APPROVED** [review]

**Pull request merged**